probe_new.py

```diff
@@ -0,0 +1,234 @@
+import asyncio
+import time
+from flask import Flask
+# import helper
+from prometheus_client import Gauge, CollectorRegistry, generate_latest
+import json
+import pykafka
+import logging
+import os
+
+LOGGER = logging.getLogger(__name__)
+
+# the flask app
+app = Flask(__name__)
+
+# create the Gauge object to track the offset change
+registry = CollectorRegistry()
+producerOffsetTracker = Gauge('kafka_offset', 'the latest available offsets', ['service', 'topic', 'partition'],
+                              registry=registry)
+consumerGroupOffsetTracker = Gauge('cg_kafka_offset', 'the consumer group offsets',
+                                   ['service', 'topic', 'partition', 'consumergroup'], registry=registry)
+
+offset_info = {}
+kafka_hosts = "c1-26-usab1-kafka-1.oracle.callstats.io:9092"
```

Comment:
This should be read from environment

Reply:
probe_new is not required anymore. Will delete the file. It was kept for testing purpose.